Source/Functions.m

```diff
@@ -85,6 +94,29 @@
 	  NSLog (_(@"Cannot load the main model file '%@'"), mainModelFile);
 	}
     }
+  else
+    {
+      mainModelFile = [infoDict objectForKey: @"NSMainStoryboardFile"];
+      if (mainModelFile != nil && [mainModelFile isEqual: @""] == NO)
+        {
+          NSStoryboard *storyboard = [NSStoryboard storyboardWithName: mainModelFile
+                                                               bundle: [NSBundle mainBundle]];
+          if (storyboard == nil)
+            {
+              NSLog (_(@"Cannot load the main storyboard file '%@'"), mainModelFile);
+            }
+          else
+            {
+              [NSStoryboard _setMainStoryboard: storyboard];
+              [storyboard _instantiateApplicationScene];
+              [storyboard instantiateInitialController];
+            }
+        }
+      else
+        {
+          NSLog(_(@"Could not find model to load %@"), mainModelFile);
```

Comment:
Shouldn't this error message talk about story board and in the current code the displayed value will always either be nil or empty.